04/Evgeny Vasiukevich/src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ChangeEvent, useEffect, useRef, useState } from 'react';
+import ApiService from './API/ApiService';
+import HomePage from './components/HomePage/HomePage';
+import HotelsBlock from './components/Hotels/HotelsBlock';
+
+function App() {
```

Comment:
Sorry, I remember about your notice to take out this in another component. But currently I don't know which component and what will be there...
Can you give me advice about it please?